Change all require statement to import (exclude any require statement… (#1928)
Shopify/quilt: packages/polyfills/CHANGELOG.md

<!-- ## Unreleased -->

### Fixed

> - Replace `require` statement with `import` statement (except for the jest polyfills wrap in if statement) so we get a proper stric ESM module.

Reviewer: ```suggestion
- Replace `require` statements with `import` statements (except for the conditional Jest polyfills) so we get a proper strict ESM module.
```
